src/main/java/se/bjurr/gitchangelog/api/helpers/Helpers.java

```diff
@@ -209,8 +209,15 @@ public static Map<String, Helper<?>> getAll() {
     helpers.put(
         "eachCommitFooter",
         (final Commit commit, final Options options) -> {
-          return each(
-              options, ConventionalCommitParser.getMessageParts(commit.getMessage()).footers);
+          if (options.hash.containsKey("matching")) {
+            final String regex = options.hash("matching");
```

Comment:
I think a better name for `matching` is `tokenMatching`. So that there can also be `valueMatching`.